bundles/binding/org.openhab.binding.comfoair/README.md

```diff
@@ -1,458 +1,1007 @@
-        var Number newLevel = 3
-
-        if( !isNight ){
-
-            var Number raumTemperatur = (Lueftung_Ablufttemperatur.state as DecimalType)
-            var Number aussenTemperatur = (Lueftung_Aussenlufttemperatur.state as DecimalType)
-            var Number zielTemperatur = (Lueftung_Komfortemperatur.state as DecimalType)
-
-            if(
-                raumTemperatur >= zielTemperatur
-                &&
-                aussenTemperatur >= raumTemperatur
-            ){
-
-                newLevel = 1
-            }
-            else if(
-                raumTemperatur >= zielTemperatur - 1
-                &&
-                aussenTemperatur >= raumTemperatur - 1
-                &&
-                currentLevel == 1
-            ){
-
-                newLevel = 1
-            }
-        }
-        else if( Fenster_OG_Schlafzimmer.state == CLOSED || Fenster_OG_Ankleide.state == OPEN ) {
-
-            newLevel = 2
-        }
-
-        if( newLevel != currentLevel ){
-
-            if( newLevel == 1 ){
-
-                logInfo("airflow.rules", "auto slowdown start")
-                    send("test@gmail.com", "Lüftung verlangsamt")
-            }
-            else if( currentLevel == 1 ){
-
-                    logInfo("airflow.rules", "auto slowdown end")
-                send("test@gmail.com", "Lüftung wieder normal")
-            }
-
-            autoChangeInProgress=true
-            sendCommand(Lueftung_Fan_Level,newLevel)
-        }
-    }
-end
-```
+# ComfoAir Binding
+
+The ComfoAir binding allows monitoring and control of a Zehnder ComfoAir 350 balanced ventilation system.
+
+This binding was tested with the Zehnder ComfoAir 350 device. Most commands should work with: ComfoAir 550, WHR930 of StorkAir, G90-380 by Wernig and Santos 370 DC to Paul.
+
+## Prerequisites
+
+Computer communication with the ComfoAir unit is carried out via the RS232 port on the device. You can use the PC port:
+- raspberry pi: UART port with converter RS232 to UART (3.3V)
+- raspberry pi: USB port with converter RS232 to USB
+- PC: RS232 port
+- PC: USB port with converter RS232 to USB
+
+The connection should be made with a 3-wire cable connecting pins: GND, TX, RX of RS232 sockets, but RX and TX pins should be crossed (TX of ComfoAir to RX of PC, RX of ComfoAir to TX of PC).
+
+## Binding Configuration
+
+You can configure this binding in the file `services/comfoair.cfg`.
+
+| Property | Default | Required | Description |
+|----------|---------|:--------:|-------------|
+| port     |         |   Yes    | Serial port which is connected to the Zehnder ComfoAir system, for example `/dev/ttyS0` on Linux or `COM1` on Windows |
+| refresh  | 60000   |   No     | refresh inverval in milliseconds
+
+### Example
+```
+port=/dev/ttyS0
+```
+
+## Item Configuration
+
+The syntax of the binding configuration strings accepted is the following:
+```
+comfoair="<device-command>"
+```

```

Comment:
Ending code fences should have a blank line following them.

Reply:
Done.